scss/_base_fontfaces.scss

```diff
@@ -1,5 +1,5 @@
 @mixin vf-b-typography-fontfaces {
-  @if str-index($font-base-family, 'Ubuntu') {
+  @if str-index($font-base-family + '', 'Ubuntu') {
```

Comment:
This seems to do the trick (casting font stack list to string), but I'm not sure if I like that. Seems like a hack.